NuGet.Config

```diff
@@ -2,13 +2,13 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="Repox Artifactory (proxy)" value="https://repox.jfrog.io/artifactory/api/nuget/nuget" />
+    <add key="Repox" value="https://repox.jfrog.io/artifactory/api/nuget/nuget" />
```

Comment:
Please rebase and update also the other NuGet.config files affected by changes in #1851